iml-agent/src/action_plugins/stratagem/action_warning.rs

```diff
@@ -2,26 +2,29 @@
 // Use of this source code is governed by a MIT-style
 // license that can be found in the LICENSE file.
 
-use crate::agent_error::ImlAgentError;
+use crate::{agent_error::ImlAgentError, env, http_comms::crypto_client};
 use csv;
+use futures::future::Future;
+use futures::stream::Stream;
 use libc;
 use std::ffi::CStr;
 use std::io;
+use std::sync::mpsc::channel;
```

Comment:
This must come from the futures crate:

```suggestion
use futures::sync::mpsc::channel;
```